package.json

```diff
@@ -7,16 +7,18 @@
     "url": "git+https://github.com/danawoodman/react-fontawesome.git"
   },
   "config": {
-    "entry": "src/index.js"
+    "entry": "src",
+    "output": "lib",
+    "mainFile": "src/index.js"
   },
   "bugs": {
     "url": "https://github.com/danawoodman/react-fontawesome/issues"
   },
   "main": "lib/index.js",
   "scripts": {
-    "build": "babel -o $npm_package_main $npm_package_config_entry",
+    "build": "babel $npm_package_config_entry -o $npm_package_config_output",
```

Comment:
Awesome, thanks! I'm curious, is it using your system babel or the one in node_modules? Confused why it was working for you but failing on my end